flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -95,6 +95,15 @@ enum class ExtremumBehavior {
   // possible to implement it without some target dependent runtime.
 };
 
+fir::ExtendedValue Fortran::lower::getAbsentIntrinsicArgument() {
+  return fir::UnboxedValue{};
+}
+
+/// Helper to test if an intrinsic argument is absent.
+static bool isAbsent(const fir::ExtendedValue &exv) {
+  return exv.getUnboxed() && !*exv.getUnboxed();
```

Comment:
`!fir::isUnboxedValue(exv)`?

Reply:
I went for `!fir::getBase(exv)`, it is even simpler.

Reply:
OK. The difference being whether the extended value has additional property values or not.